### PR TITLE
Now is possible use other attribute to ignore field without lost the …

### PR DESCRIPTION
**Pull request by rhvieira1980**
…json serialization option in the POCO.

resolved by: #1

### CODE REVIEW

File: src/Parquet/Serialization/Attributes/ParquetIgnoreFieldAttribute.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace Parquet.Serialization.Attributes {
+
+    /// <summary>
+    /// Attribute to Ignore Field in Serialization, some times need data in json but not need in parquet.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class ParquetIgnoreFieldAttribute : Attribute {
```
Comment (aloneguid) [inline src/Parquet/Serialization/Attributes/ParquetIgnoreFieldAttribute.cs:11]: As we already have `[ParquetRequired]` it's probalby better to name this `ParquetIgnoreAttribute`?

**Review by aloneguid — CHANGES_REQUESTED** [review]

**Comment by aloneguid** [comment]
You might also want to add a unit test for this attribute to avoid it being broken in future.

**Comment by rhvieira1980** [comment]
In this case the attribute is made to Ignore, not to require.

This usually use if you want to serialize the attribute in json, but not in parquet .

Best

________________________________
De: Ivan G ***@***.***>
Enviado: quarta-feira, 22 de maio de 2024 08:16
Para: aloneguid/parquet-dotnet ***@***.***>
Cc: Rafael Vieira ***@***.***>; Author ***@***.***>
Assunto: Re: [aloneguid/parquet-dotnet] Now is possible use other attribute to ignore field without lost the … (PR #511)


@aloneguid requested changes on this pull request.

________________________________

In src/Parquet/Serialization/Attributes/ParquetIgnoreFieldAttribute.cs<https://github.com/aloneguid/parquet-dotnet/pull/511#discussion_r1609765533>:

> @@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Parquet.Serialization.Attributes {
+
+    /// <summary>
+    /// Attribute to Ignore Field in Serialization, some times need data in json but not need in parquet.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    public class ParquetIgnoreFieldAttribute : Attribute {


As we already have [ParquetRequired] it's probalby better to name this ParquetIgnoreAttribute?

File: src/Parquet/Serialization/Attributes/ParquetIgnoreFieldAttribute.cs (hunk 1 above)
Comment (rhvieira1980) [reply src/Parquet/Serialization/Attributes/ParquetIgnoreFieldAttribute.cs:11]: ok, Now I understand that you said

**Review by aloneguid — APPROVED** [review]
Looks fine, thanks! It will be included in the current pre-release version.

**Pull request merged**